src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRefExtended.java

```diff
@@ -0,0 +1,32 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Optional;
+
+import java.util.function.Function;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Objects; // violation 'Unused import - java.util.Objects.'
+import static java.util.Arrays.asList; // violation 'Unused import - java.util.Arrays.asList.'
+
+public class InputUnusedImportsFromStaticMethodRefExtended {
```

Comment:
Please add a few fields that use method references in here as well.

Reply:
@nrmancuso done